tests/performance/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.8'
 
 services:
   master:
-    image: local/locust:latest
+    image: itisfoundation/locust:${LOCUST_VERSION}
```

Comment:
are you sure about the `itisfoundation` as a prefix for images?

Reply:
yes